lib/functions/function_plot_measures.R

```diff
@@ -0,0 +1,31 @@
+plot_measures <- function(
+    data,
+    title = NULL,
+    x_label = NULL,
+    y_label = NULL) {
+
+  # Create plot
+  plot <- ggplot(
+    data,
+    aes(
+      x = interval_end,
+      y = numerator,
+      color = measure,
+      group = measure
+    )
+  ) +
+    geom_line() +
+    labs(
+      title = title,
+      x = x_label,
+      y = y_label,
+      color = "Condition"
+    ) +
+    # Setting the minimum y-value
+    ylim(y_min, NA) +
+    # Applying the minimal theme
+    theme_minimal()
```

Comment:
What do you think about adding a vertical line to indicate the Pharmacy First start date? For example with something like this:

```suggestion
    theme_minimal() +
    geom_vline(
      xintercept = lubridate::as_date("2024-02-01")),
      linetype = "dotted",
      colour = "orange",
      size = .7
    )
```

Reply:
Will address this in the next commit.